Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -556,7 +556,7 @@ void ProgramShaderCache::CreateHeader()
 
 		, GetGLSLVersionString().c_str()
 		, v<GLSL_140 ? "#extension GL_ARB_uniform_buffer_object : enable" : ""
-		, g_ActiveConfig.backend_info.bSupportsEarlyZ ? "#extension GL_ARB_shader_image_load_store : enable" : ""
+		, v<GLSLES_300 && g_ActiveConfig.backend_info.bSupportsEarlyZ ? "#extension GL_ARB_shader_image_load_store : enable" : ""
```

Comment:
Err, I'd prefer adding a bool "isGLSLES", and using that in more places; using less-than/greater-than on GLSL_VERSION to implicitly specify either desktop GL or GLES is really confusing.